sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -3471,121 +2981,110 @@ public partial class SearchResultsPage<T> : Azure.Page<Azure.Search.Documents.Mo
+        public static implicit operator Azure.Search.Documents.Models.SemanticErrorReason (string value) { throw null; }
+        public static bool operator !=(Azure.Search.Documents.Models.SemanticErrorReason left, Azure.Search.Documents.Models.SemanticErrorReason right) { throw null; }
         public override string ToString() { throw null; }
     }
-    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
-    public readonly partial struct SemanticPartialResponseReason : System.IEquatable<Azure.Search.Documents.Models.SemanticPartialResponseReason>
+    public partial class SemanticSearchOptions
     {
-        private readonly object _dummy;
-        private readonly int _dummyPrimitive;
-        public SemanticPartialResponseReason(string value) { throw null; }
-        public static Azure.Search.Documents.Models.SemanticPartialResponseReason CapacityOverloaded { get { throw null; } }
-        public static Azure.Search.Documents.Models.SemanticPartialResponseReason MaxWaitExceeded { get { throw null; } }
-        public static Azure.Search.Documents.Models.SemanticPartialResponseReason Transient { get { throw null; } }
-        public bool Equals(Azure.Search.Documents.Models.SemanticPartialResponseReason other) { throw null; }
-        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
-        public override bool Equals(object obj) { throw null; }
-        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
-        public override int GetHashCode() { throw null; }
-        public static bool operator ==(Azure.Search.Documents.Models.SemanticPartialResponseReason left, Azure.Search.Documents.Models.SemanticPartialResponseReason right) { throw null; }
-        public static implicit operator Azure.Search.Documents.Models.SemanticPartialResponseReason (string value) { throw null; }
-        public static bool operator !=(Azure.Search.Documents.Models.SemanticPartialResponseReason left, Azure.Search.Documents.Models.SemanticPartialResponseReason right) { throw null; }
-        public override string ToString() { throw null; }
+        public SemanticSearchOptions() { }
+        public Azure.Search.Documents.Models.QueryAnswer QueryAnswer { get { throw null; } set { } }
+        public Azure.Search.Documents.Models.QueryCaption QueryCaption { get { throw null; } set { } }
+        public string SemanticConfigurationName { get { throw null; } set { } }
+        public Azure.Search.Documents.Models.SemanticErrorMode? SemanticErrorMode { get { throw null; } set { } }
+        public System.TimeSpan? SemanticMaxWaitInMilliseconds { get { throw null; } set { } }
+    }
+    public partial class SemanticSearchResult
+    {
+        public SemanticSearchResult() { }
+        public System.Collections.Generic.IList<Azure.Search.Documents.Models.CaptionResult> Captions { get { throw null; } }
```

Comment:
Since this is a response only model, change type to `IReadOnlyList<CaptionResult>`.